.vscode/settings.json

```diff
@@ -1,7 +1,5 @@
 {
   "editor.formatOnSave": true,
-  "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "prettier.configPath": ".eslintrc.js",
```

Comment:
Can you rather add the simple comma `'` rule instead.

See link: https://prettier.io/docs/en/options.html#quotes

What this line does is to implement the default `.eslintrc.js` linting rules, this can be extended and editted in the root `.eslintrc.js`

See: https://eslint.org/docs/latest/use/configure/configuration-files

Reply:
This is still pending, I'm focused on other things so I haven't been able to review it yet